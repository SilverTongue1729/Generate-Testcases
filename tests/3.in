5
3
1 1 234872
3
1 1 234872
4
100 1 1 200
7
5 1 1 1 1 14 15
21
5 5 5 5 5 5 5 5 5 5 1 1 1 1 2 5 20 1 1 1 1000000