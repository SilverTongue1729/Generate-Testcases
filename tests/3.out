234874
234874
202
19
1000005
